AnyText/AnyText.Core/Matcher.cs

```diff
@@ -58,27 +58,77 @@ public IEnumerable<RuleApplication> GetRuleApplicationsAt(ParsePosition position
             var line = _memoTable[position.Line];
             foreach (var col in line.Columns)
             {
-                if (col.Key > position.Col)
+                //TODO: Nur dann überspringen, wenn dazwischen etwas anderes als ein Whitespace (implementiert, aber testen)
+                if (col.Key > position.Col && String.IsNullOrWhiteSpace(context.Input[position.Line].Substring(position.Col, Math.Min(context.Input[position.Line].Length, col.Key - position.Col))))
```

Comment:
die Länge der Zeile als Länge des Substrings zu nehmen kommt mir komisch vor, falls `position.Col > 0` geht das in eine Exception